.eslintrc.json

```diff
@@ -3,7 +3,9 @@
     "plugins": [
         "@typescript-eslint",
         "header",
-        "deprecation"
+        "deprecation",
```

Comment:
Tell me if I am going overboard here, but should we consider sorting this list of plugins and also the parserOptions? The parserOptions list in particular is getting quite long and it might be good to bring some structure to it.

Reply:
For some of these config options the order matters, although I can't remember which. For example if things conflict with each other (i.e. eslint and prettier might format things a little different) the one listed last is the one that takes effect

Btw, I think you mean the "rules" section, not the "parserOptions" section. The file is only 64 lines long, so I'm not worried about it 😆 You're welcome to organize it

Reply:
Yes, you're right. I meant the "rules" section. The diff made it seem like the "rules" were under "parserOptions". I can sort them once you merge this PR.